fiddler/datadog_checks/fiddler/data/conf.yaml.example

```diff
@@ -0,0 +1,69 @@
+## All options defined here are available to all instances.
+#
+init_config:
+
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
+instances:
+
+    ## @param url - string - required
+    ## The Fiddler end point URL
+    #
+  - url: https://demo.trial.fiddler.ai
```

Comment:
Can you please run `ddev -x validate config -s fiddler` so that this file can be updated with the values from the `spec.yaml` file?

Reply:
Done.